### ISSUE TITLE
No way to Donate to the individuals who are working so hard on this project during this epidemic

### ISSUE THREAD

**Issue by Noe007** [CSSEGISandData/COVID-19#375]
You guys are working hard, everyone is using this tool to monitor the situation.

**Comment by jwa5426** [comment]
I agree that this is an awesome tool and we should all appreciate and support the efforts of the team maintaining it.

I’m genuinely curious though, was this put together by volunteers? Or is it a non-profit effort? Sorry if this is an ignorant question, I really don’t know.

**Comment by CSSEGISandData** [comment]
Thank you! We appreciate the kind words and we are glad you are enjoying the dashboard!
